client/src/components/Pages/Help/questions.js

```diff
@@ -0,0 +1,55 @@
+const homepage = {
+  title: 'Home Page',
+  children: [
+    { title: 'What is your name?' },
```

Comment:
Probably if there are no other levels, I could use it as just sting, no need to create an object. But I was thinking that it will be more recursive, and to ensure that I can easily add more levels, I decided to put it this way. Now I see that it makes no sense and it is better to change it.